src/main/java/org/jitsi/jicofo/conference/JitsiMeetConferenceImpl.java

```diff
@@ -1666,13 +1655,22 @@ private void onBridgeUp(Jid bridgeJid)
     }
 
     /**
-     * Handles the case of some bridges in the conference becoming non-operational.
-     * @param bridges the bridges that are non-operational.
+     * Handles the case of some bridges in the conference becoming non-operational. The bridge is assumed to still be
+     * used by {@link #colibriSessionManager}
+     * @param bridge the bridges to stop using.
      */
-    private void onMultipleBridgesDown(Set<Bridge> bridges)
+    private void onBridgeDown(Bridge bridge)
     {
-        List<String> participantIdsToReinvite = colibriSessionManager.removeBridges(bridges);
+        List<String> participantIdsToReinvite = colibriSessionManager.removeBridge(bridge);
+        if (!participantIdsToReinvite.isEmpty())
+        {
+            listener.bridgeRemoved(1);
```

Comment:
Why does the bridge count only change if the bridge had participants?

Reply:
The purpose of the "bridges removed" stat was to show how many conferences were affected.

Reply:
It seems a bit confusingly named then - how is it shown when it's exposed?

Reply:
It's bridges_removed: https://github.com/jitsi/jicofo/blob/master/src/main/java/org/jitsi/jicofo/FocusManager.java#L431

I agree it's confusing. Do you have suggestions for a better name? The changes should go in a separate PR though

Reply:
conferences_removed_from_bridges?  Or is it counting the number of bridges that had conferences removed from them?

Reply:
I'll just remove it, it's redundant given participants_moved.